foster55f/Hotel-Overlook
# Class fetch

What's this PR do?
This PR update created the 2 js files to be attached to the Manager.html and User.html files.

Where should the reviewer start?
The 2 new js files being used for the 2 html files

How should this be manually tested?
I would pull the repo down and console.log to make sure data is being pulled correctly

Any background context you want to provide?
This Repo class will now be built out
